.github/workflows/build-test.yml

```diff
@@ -69,21 +71,21 @@ jobs:
     needs: pre-flight-check
     steps:
       - name: Setup ROS
-        uses: ros-tooling/setup-ros@0.2.1
+        uses: ros-tooling/setup-ros@v0.3
         with:
           required-ros-distributions: foxy
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - run: ./tools/ci/run-integration-test.sh
 
   build-test-ubuntu-with-sanitizers-gcc-latest:
     runs-on: ubuntu-20.04
     needs: pre-flight-check
     steps:
-      - uses: egor-tensin/setup-gcc@v1.2
+      - uses: egor-tensin/setup-gcc@v1
```

Comment:
Are we sure that we want to downgrade here?

Reply:
@mossmaurice 
The versioning for Github Action is a bit different. The Github Action maintainer sets always a git tag with a major version only e.g. `v1` and add then subsequently new git tags with bugfixes e.g. `v1.1` and `v1.2`
The `v1` tag is always relocated to the latest bugfix (v1.2 in this case).

Comparing v1 and v1.2 tag will show that they are the same: https://github.com/egor-tensin/setup-gcc/compare/v1...v1.2

With that we get automatically all bugfixes for a Github action.